docs/cep/index.md

```diff
@@ -22,11 +22,115 @@ You can process streams to perform the following actions with your data:
 - Transform data from one format to another. For example, from XML to JSON.
```

Comment:
We don't support XML. How about text to JSON? Or really, anything in Sourcemapper to anything in Sinkmapper.